pkg/policy/existing.go

```diff
@@ -34,8 +34,13 @@ func (pc *PolicyController) processExistingResources(policy kyverno.ClusterPolic
 		}
 
 		// skip reporting violation on pod which has annotation pod-policies.kyverno.io/autogen-applied
-		if skipPodApplication(resource, logger) {
-			continue
+		ann := resource.GetAnnotations()
```

Comment:
We should check the annotation on Policy, not resource.

Reply:
Good catch @realshuting!